libraries/esp8266/examples/NTP-TZ-DST/NTP-TZ-DST.ino

```diff
@@ -212,8 +213,18 @@ void setup() {
 
   // NTP servers may be overriden by your DHCP server for a more local one
   // (see below)
+
+  // ----> Here is the ONLY ONE LINE needed in your sketch
+
   configTime(MYTZ, "pool.ntp.org");
 
+  //       Here is the ONLY ONE LINE needed in your sketch <----
+  // pick a value from TZ.h (search for this file in your filesystem) for MYTZ
+
+  // former configTime is still valid, here is the call for 7 to the west
```

Comment:
If this is no longer preferred, maybe drop completely from demo?  Don't need to show *all* ways, just the *recommended* ones IMHO.

Reply:
The example shows what should be used, but also tells the former API is still good for users wanting to upgrade but also seeking for compatibility with other APIs (esp32).

Thanks for the review, all other points are addressed I guess.